lesson-5/Chen_Homework Source Files/style.css

```diff
@@ -0,0 +1,38 @@
+body {
+  font-family: "Work Sans", sans-serif;
+}
+.container {
+	position:fixed;
+	top:50%;
+	left:50%;
+	transform: translate(-50%,-50%);
+}
+.colorBox{
+	background-color: #f7f2f2;
+	
+	padding:20px;
+	width:235px;
+
+}
+
+a{background-color:white;
```

Comment:
it's easier to read this if you start the properties on the line after the opening bracket (like you did on line 4-5)